src/main/java/com/alphatica/genotick/data/MainAppData.java

```
@@ -113,4 +113,12 @@ public boolean containsDataSet(DataSetName name) {
     public boolean isEmpty() {
         return dataSetMap.isEmpty();
     }
+    
+    public int getMaximumColumnCount() {
+    		int maximumColumnCount = 0;
+    		for(DataSet dataSet : dataSetMap.values()) {
+    			maximumColumnCount = Math.max(maximumColumnCount,  dataSet.getColumnCount());
+    		}
+    		return maximumColumnCount;
+    }
```

Too big indent. Also this could be just
`return dataSetMap.values().stream().mapToInt(DataSet::getColumnCount).max().orElseThrow();` 
with throwing something. Or get 0, but throw fits 'fail fast' philosophy.